### ISSUE TITLE
TypeError: 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by raphtlw** [InstaPy/InstaPy#6206]
## Expected Behavior
It should work normally

## Current Behavior
It crashes with an error:
```shell
Traceback (most recent call last):
  File "/home/raphael/Desktop/Projects/inspiredtoshoot/instagram-bot/src/main.py", line 69, in <module>
    session.follow_user_followers(
  File "/home/raphael/Desktop/Projects/inspiredtoshoot/instagram-bot/.venv/lib/python3.9/site-packages/instapy/instapy.py", line 3749, in follow_user_followers
    followed = self.follow_by_list(
  File "/home/raphael/Desktop/Projects/inspiredtoshoot/instagram-bot/.venv/lib/python3.9/site-packages/instapy/instapy.py", line 1271, in follow_by_list
    self.interact_by_users(
  File "/home/raphael/Desktop/Projects/inspiredtoshoot/instagram-bot/.venv/lib/python3.9/site-packages/instapy/instapy.py", line 2511, in interact_by_users
    links = get_links_for_username(
  File "/home/raphael/Desktop/Projects/inspiredtoshoot/instagram-bot/.venv/lib/python3.9/site-packages/instapy/like_util.py", line 476, in get_links_for_username
    is_private = is_private_profile(browser, logger, following_status == "Following")
  File "/home/raphael/Desktop/Projects/inspiredtoshoot/instagram-bot/.venv/lib/python3.9/site-packages/instapy/util.py", line 82, in is_private_profile
    is_private = shared_data["entry_data"]["ProfilePage"][0]["graphql"]["user"][
TypeError: 'NoneType' object is not subscriptable
```

## InstaPy configuration
```python
session.set_do_like(enabled=True, percentage=70)
session.set_user_interact(amount=5, randomize=True, percentage=70, media="Photo")

accounts = [
    "moodygrams",
    "visualsofearth",
    "artofvisuals",
    "illgrammers",
    "citykillerz",
    "shotzdelight",
]
session.follow_user_followers(
    accounts,
    amount=100,
    randomize=True,
    interact=True,
)
session.follow_likers(
    accounts,
    photos_grab_amount=2,
    follow_likers_per_photo=10,
    randomize=True,
    sleep_delay=600,
    interact=True,
)
session.unfollow_users(
    amount=126,
    instapy_followed_enabled=True,
    instapy_followed_param="all",
    style="RANDOM",
    unfollow_after=432000,  # Unfollow after 5 days
    sleep_delay=800,
)
```

**Comment by schealex** [comment]
possibly related to #6195 and fixed by PR #6195. Can you please check if that PR fixes your issue too and leave feedback?

**Comment by raphtlw** [comment]
Looks like it fixed it, thanks.
